.coveragerc

```diff
@@ -1,4 +1,26 @@
 [run]
+branch = True
+source = .
 omit =
-    # standlonetemplate is read dynamically and tested by test_genscript
-    *standalonetemplate.py
+    .tox/*
+    /usr/*
+    setup.py
+
+[report]
+show_missing = True
+skip_covered = True
+exclude_lines=
+    # Have to re-enable the standard pragma
+    \#\s*pragma: no cover
+
+    # Don't complain if tests don't hit defensive assertion code:
+    ^\s*raise AssertionError\b
+    ^\s*raise NotImplementedError\b
+    ^\s*return NotImplemented\b
+    ^\s*raise$
+
+    # Don't complain if non-runnable code isn't run:
+    ^if __name__ == ['"]__main__['"]:$
+
+    # Intentionally skipped tests
+    ^\s*@pytest\.mark\.skipif
```

Comment:
`@pytest\.mark\.skipif` should not be excluded: one of the builds should hit it.
(more relevant with/after #2800)

Reply:
I don't see a reason to block on #2800 -- until then this improves local runs of isolated environments.

Reply:
Yes, it is not blocked by #2800, but with #2800 you would not have added it in the first place, would you? (because it should be covered then, and if not it would mean no test setup is hitting it).